evap/staff/tests/test_views.py

```diff
@@ -1958,6 +1963,19 @@ def test_evaluation_create(self):
         form.submit()
         self.assertEqual(Evaluation.objects.get().name_de, "lfo9e7bmxp1xi")
 
+    def test_evaluation_create_main_language(self):
+        response = self.app.get(self.url_for_semester, user=self.manager, status=200)
+        form = response.forms["evaluation-form"]
+        self._set_valid_form(form)
+
+        with self.assertRaises(ValueError):
+            form["main_language"] = "some_wrong_value"
+            form.submit()
```

Comment:
Same here, the second line is never reached
```suggestion
        with self.assertRaises(ValueError):
            form["main_language"] = "some_wrong_value"
```